drivers/sensor/CMakeLists.txt

```diff
@@ -58,7 +58,6 @@ add_subdirectory_ifdef(CONFIG_INA3221 ina3221)
 add_subdirectory_ifdef(CONFIG_IST8310 ist8310)
 add_subdirectory_ifdef(CONFIG_ISL29035 isl29035)
 add_subdirectory_ifdef(CONFIG_ISM330DHCX ism330dhcx)
-add_subdirectory_ifdef(CONFIG_ITDS wsen_itds)
```

Comment:
Please explain in the commit message why you're removing the driver

Reply:
Would like to see more explanation than just "we're removing the driver to replace it with a new one". You mention the reason in https://github.com/zephyrproject-rtos/zephyr/pull/62408#issuecomment-1711566468, but would like to have that captured in the commit history.

Reply:
i changed the commit message to indicate that the new hal is not compatible with the old drivers.